bot.go

```diff
@@ -265,16 +265,19 @@ func eval(input string, formatMode, compactMode, verbatimMode bool) string {
 	return res
 }
 
+// Discord's limit - some margin  for adding we are truncating, in characters/runes.
+const MaxMessageLength = 2000 - 100
+
 // returns the id of the reply.
 func evalAndReply(session *discordgo.Session, info, channelID, input string,
 	replyID string, formatMode, compactMode, verbatimMode bool,
 ) string {
 	res := eval(input, formatMode, compactMode, verbatimMode)
 	level := log.Info
 	msg := "response"
-	if len(res) > 1900 {
-		// hope we didn't garbage some unicode char in the middle of a utf8 char...
-		res = res[:1900] + fmt.Sprintf("```...truncated from %d...", len(res))
+	runes := []rune(res)
+	if len(runes) > MaxMessageLength {
+		res = string(runes[:1900]) + fmt.Sprintf("```...truncated from %d (%d bytes)...", len(runes), len(res))
```

Comment:
I don't get the triple backtick. is there always a  triple backtick at the end of the message the message?

Reply:
almost always, unless someone uses the hidden `-v` flag (verbatim mode)

and I checked in that case it silently truncate:


<img width="1000" alt="Screenshot 2024-08-16 at 4 44 43 PM" src="https://github.com/user-attachments/assets/bf38da7b-8d57-451d-b047-c44825703cbc">


ps: you should come to discord to play with the bot :)